modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/IConfigurationPropertyStoreService.java

```diff
@@ -147,6 +147,26 @@ String getProperty(@NotNull String prefix, @NotNull String suffix, String... inf
      */
     void deleteProperty(@NotNull String name) throws ConfigurationPropertyStoreException;
 
+    /**
+     * <p>
+     * deletePrefixedProperties will delete all the properties from the standard Configuration Property Store starting the with provided prefix in an atomic operation.
+     * </p>
+     * 
+     * <p>
+     * As an example, if we called deletePrefixedProperties("test.stream.mystream.") within the a
+     * namespace, then all the properties starting with "test.stream.mystream." will be deleted
+     * </p>
+     * 
+     * <p>
+     * If a property could not be deleted, a ConfigurationPropertyStoreException is thrown.
+     * </p>
+     * 
+     * @param prefix The prefix for a property.
+     * @throws ConfigurationPropertyStoreException
+     * @throws FrameworkPropertyFileException 
```

Comment:
Minor - the `<p>` tags aren't needed, and we could add a reason why a `ConfigurationPropertyStoreException` can be thrown. I don't think we need `FrameworkPropertyFileException` to be thrown also